mungegithub/mungers/matchers/comment/command.go

```diff
@@ -34,25 +34,26 @@ var (
 	// - Line that starts with slash
 	// - followed by non-space characteres,
 	// - (optional) followed by space and arguments
-	commandRegex = regexp.MustCompile(`^/([^\s]+) *?([^\n]*)`)
+	commandRegex = regexp.MustCompile(`(?m)^/([^\s]+)[\t ]*([^\n\r]*)`)
```

Comment:
I don't speak regex.  @spxtr does this make sense to you/look similar to the prow one?